package.json

```diff
@@ -80,7 +80,7 @@
     "@types/global-agent": "^2.1.0",
     "applicationinsights": "^1.5.0",
     "applicationinsights-js": "^1.0.20",
-    "axios": "^0.18.0",
+    "axios": ">=0.21.1",
```

Comment:
Why is this `>=0.21.1` while the other projects have it as `^0.21.1`?

Reply:
there were some high vulnerabilities. after running 'yarn audit --groups dependencies' it suggested >=0.21.1. we confirmed with uday the version number. he is adding the same on his AO feature branch.

Reply:
Understood. Then we should presumably do the same on MC, MO, and any other projects that use Axios.